### PR TITLE
Mic 1211

### PR DESCRIPTION
**Pull request by ghost**
Implementation of https://spheremall.atlassian.net/wiki/spaces/MIC/pages/1304592428/Grapher+2.3.7+Release+Notes

### CODE REVIEW

File: src/Lib/Makers/ObjectMaker.php [hunk 1]
```diff
@@ -135,14 +135,15 @@ protected function getRelationships(array $item, array $included)
      * result example: ['brands'][706] = {attributes}
      *
      * @param array $included
+     * @param bool  $getAttributes
      *
      * @return array
      */
-    protected function getIncludedArray(array $included)
+    protected function getIncludedArray(array $included, $getAttributes = true)
```
Comment (say-my-name-user) [inline src/Lib/Makers/ObjectMaker.php]: add `bool` type to parameter declaration

File: src/Lib/Mappers/EntitiesCorrelationMapper.php [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+ * Class EntityCorrelationMapper
+ *
+ * @package SphereMall\MS\Lib\Mappers
+ */
+class EntitiesCorrelationMapper extends Mapper
+{
+    /**
+     * @param array $array
+     *
+     * @return mixed
```
Comment (say-my-name-user) [inline src/Lib/Mappers/EntitiesCorrelationMapper.php]: `@return null` ?

File: src/Resources/Grapher/CorrelationsResource.php [hunk 3]
```diff
@@ -96,4 +73,45 @@ public function delete($id)
...
+     *
+     * @return array|Collection
+     * @throws \GuzzleHttp\Exception\GuzzleException
+     */
+    public function getById(int $id, string $forClassName)
+    {
+        $params = $this->getQueryParams();
+
+        $type      = CorrelationTypeHelper::getGraphTypeByClass($forClassName);
+        $uriAppend = "{$type}/{$id}";
```
Comment (say-my-name-user) [inline src/Resources/Grapher/CorrelationsResource.php:89]: `$uriAppend` variable is used only once

File: src/Resources/Grapher/CorrelationsResource.php [hunk 4]
```diff
@@ -96,4 +73,45 @@ public function delete($id)
...
+     * @param array  $entityIds
+     *
+     * @param array  $filterParams
+     *
+     * @return array|int|Entity|Collection
+     * @throws \GuzzleHttp\Exception\GuzzleException
+     */
+    public function getFromEntityByIds(string $entityFrom, array $entityIds, array $filterParams = [])
+    {
+        $uriAppend = "from/{$entityFrom}";
```
Comment (say-my-name-user) [inline src/Resources/Grapher/CorrelationsResource.php:107]: `$uriAppend` variable is used only once

File: src/Resources/Grapher/CorrelationsResource.php (hunk 3 above)
Comment (ghost) [reply src/Resources/Grapher/CorrelationsResource.php:89]: This is templated format (all resources functions is written like this)

**Review by say-my-name-user — APPROVED** [review]

**Review by SergiiBondarchuk — APPROVED** [review]

**Pull request merged**